Scripts/Python/xDynTextLoc.py

```diff
@@ -63,6 +63,7 @@
 clearColorG = ptAttribFloat(16, "Clear Color (Green)", default=0.0)
 clearColorB = ptAttribFloat(17, "Clear Color (Blue)", default=0.0)
 clearColorA = ptAttribFloat(18, "Clear Color (Alpha)", default=0.0)
+blockRGB = ptAttribBoolean(19, "BlockRGB")
```

Comment:
A slightly more descriptive name would be nice... something like "Support Alpha Blending" maybe?